engine/pom.xml

```diff
@@ -145,7 +145,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>2320</maxAllowedViolations>
+          <maxAllowedViolations>2063</maxAllowedViolations>
```

Comment:
The violations were fixed as part of other commits which I don't know. Whoever fixed them didn't change this number